common/changes/@rushstack/lockfile-explorer/octogonz-lfx-pnpm-10_2025-10-03-05-47.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/lockfile-explorer",
+      "comment": "Add support for PNPM 10 lockfile format",
+      "type": "major"
```

Comment:
Why not minor?

Reply:
I think we're too conservative about bumping our versions :)